index.html

```diff
@@ -21,18 +23,198 @@
         </div>
       </nav>
     </header>
+    <!-- Headline Section -->
     <section class="intro-section">
       <h1>I’m Cristian.<br>Glad to see you!</h1>
       <p>I’m a software developer! I can help you build a product, feature, or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
       <a class="connect-btn">LET'S CONNECT</a>
       <ul class="social-icons">
-        <li><a href=""><img src="assets/twitter.svg" alt="Twitter icon" class="social-icon"></a></li>
-        <li><a href=""><img src="assets/linkedin.svg" alt="LinkedIn icon" class="social-icon"></a></li>
-        <li><a href=""><img src="assets/medium.svg" alt="Medium icon" class="social-icon"></a></li>
-        <li><a href=""><img src="assets/github.svg" alt="GitHub icon" class="social-icon"></a></li>
-        <li><a href=""><img src="assets/angel.svg" alt="Angelist icon" class="social-icon"></a></li>
+        <li><a href=""><img src="assets/twitter.svg" alt="Cristian's Twitter" class="social-icon"></a></li>
+        <li><a href=""><img src="assets/linkedin.svg" alt="Cristian's LinkedIn" class="social-icon"></a></li>
+        <li><a href=""><img src="assets/medium.svg" alt="Cristian's Medium" class="social-icon"></a></li>
+        <li><a href=""><img src="assets/github.svg" alt="Cristian's GitHub" class="social-icon"></a></li>
+        <li><a href=""><img src="assets/angel.svg" alt="Cristian's Angelist" class="social-icon"></a></li>
       </ul>
     </section>
+<!-- Projects Section -->
+<section class="project-section" aria-label="Projects">
+    <!-- 1st project card -->
+    <article class="project-card"> 
+      <div class="project-img"> 
+        <img src="assets/Snapshoot-1.svg" alt="Screenshot of Project 1">
+      </div>
+      <div class="project-content"> 
+        <h2>Tonic</h2>
```

Comment:
To ensure that we stick to stick to the design as much as possible, let's use the same color for the title as Figma 👍😎

Your design
![image](https://github.com/CVILLA09/Portfolio/assets/78251927/4330145e-106b-4235-a94b-2a2f0bdbec4b)

Figma
![image](https://github.com/CVILLA09/Portfolio/assets/78251927/5c4efbe3-719f-4d4d-b803-651cd5f50111)